src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/IepLevelResourceTest.kt

```diff
@@ -735,9 +736,25 @@ class IepLevelResourceTest : SqsIntegrationTestBase() {
 
       webTestClient.post().uri("/iep/migration/booking/$bookingId")
         .headers(setAuthorisation(roles = listOf("ROLE_MAINTAIN_IEP"), scopes = listOf("write")))
-        .bodyValue(syncPostRequest("Standard")) // fails validation because it's > 6 length
```

Comment:
because the object now validates its parameters, the JSON body must be manually created to avoid checks